MPCAutofill/cardpicker/search/search_functions.py

```diff
@@ -273,11 +276,17 @@ def retrieve_cardback_identifiers(self) -> list[str]:
                 if self.languages
                 else ~Q(pk__in=[])
             )
-            tag_filter = (Q(tags__contains=self.tags) | Q(tags__contained_by=self.tags)) if self.tags else ~Q(pk__in=[])
+            includes_tag_filter = (
+                (Q(tags__contains=self.includes_tags) | Q(tags__contained_by=self.includes_tags))

```

Comment:
This would depend on if we want the include tags to be:
- at least one of the include list (like it is here). You could remove the contains and it should return the same result.
- at least all of the include list (then it would just be contains)

Reply:
yea, the intention is at least one of the specified tags to include. iirc filtering the query wasn't working like i expected when i removed the contains, but will double check after work tonight

Reply:
confirmed that changing this to just either `tags__contains` or just `tags__contained_by` causes test failures, so going to leave this as-is